File: Android/sources/com/android/billingclient/api/zzar.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

final class zzar implements Callable {
    final /* synthetic */ String zza;
    final /* synthetic */ PurchaseHistoryResponseListener zzb;
    final /* synthetic */ BillingClientImpl zzc;

    public zzar(BillingClientImpl billingClientImpl, String str, PurchaseHistoryResponseListener purchaseHistoryResponseListener) {
        this.zza = str;
        this.zzb = purchaseHistoryResponseListener;
        this.zzc = billingClientImpl;
    }

    public final /* bridge */ /* synthetic */ Object call() throws Exception {
        zzbp zzg = BillingClientImpl.zzg(this.zzc, this.zza);
        this.zzb.onPurchaseHistoryResponse(zzg.zza(), zzg.zzb());
        return null;
    }
}

File: IOS/v2/com/android/billingclient/api/zzar.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
final class zzar implements Callable {.swift
final /* synthetic */ String zza;.swift
final /* synthetic */ PurchaseHistoryResponseListener zzb;.swift
final /* synthetic */ BillingClientImpl zzc;.swift
public zzar(BillingClientImpl billingClientImpl, String str, PurchaseHistoryResponseListener purchaseHistoryResponseListener) {.swift
this.zza = str;.swift
this.zzb = purchaseHistoryResponseListener;.swift
this.zzc = billingClientImpl;.swift
}.swift
public final /* bridge */ /* synthetic */ Object call() throws Exception {.swift
zzbp zzg = BillingClientImpl.zzg(this.zzc, this.zza);.swift
this.zzb.onPurchaseHistoryResponse(zzg.zza(), zzg.zzb());.swift
return null;.swift
}.swift
}.swift
